### PR TITLE
doc and example notebook fixes

### PR DESCRIPTION
**Pull request by amnona**
fix the "WARNING: Pygments lexer name 'ipython3' is not known" warning
update and rerun the example notebooks
fix bug in gnps_ms database information display

ready for merge :)

### CODE REVIEW

File: calour/heatmap/plotgui_jupyter.py [hunk 1]
```diff
@@ -178,7 +179,9 @@ def show_info(self):
                 annt_type = details.get('annotationtype', 'None')
                 # annt_id = details.get('annotationid', 'NA')
                 ccolor = colors.get(annt_type, 'black')
+                self._ipyw_sid.value = 'pootaasdsdfg'
```
Comment (RNAer) [inline calour/heatmap/plotgui_jupyter.py]: what is `poot...`? same for line 184

File: doc/source/conf.py [hunk 2]
```diff
@@ -97,7 +97,11 @@ def fmt(string):
     'sphinx.ext.doctest',
     'sphinx.ext.autosummary',
     'sphinx.ext.intersphinx',
-    'nbsphinx'
+    'nbsphinx',
+    # this extenstion is needed to avoid the
+    # "WARNING: Pygments lexer name 'ipython3' is not known" error for the notebooks.
+    # fix based on: https://github.com/spatialaudio/nbsphinx/issues/24
+    'IPython.sphinxext.ipython_console_highlighting'
```
Comment (RNAer) [inline doc/source/conf.py]: wait... we don't have those warnings: http://biocore.github.io/calour/notebooks/microbiome_step_by_step.html

File: calour/heatmap/plotgui_jupyter.py (hunk 1 above)
Comment (amnona) [reply calour/heatmap/plotgui_jupyter.py]: oops. an old debugging artifact :)
killed

File: doc/source/conf.py (hunk 2 above)
Comment (amnona) [reply doc/source/conf.py]: i don't understand :(
if you do (in the docs):
make clean
make html

you get a lot of : "WARNING: Pygments lexer name 'ipython3' is not known" warnings
this fixes these warnings

File: calour/heatmap/plotgui_jupyter.py (hunk 1 above)
Comment (RNAer) [reply calour/heatmap/plotgui_jupyter.py]: you haven't!

Comment (amnona) [reply calour/heatmap/plotgui_jupyter.py]: yes i have 
(forgot to commit - now should be ok...)

**Comment by amnona** [comment]
ok, ready for merge :)

**Pull request merged**